### DISCUSSION TITLE
My Github Account is compromised

### DISCUSSION THREAD

**Discussion by gcs1813** [community/community#120062] [Other Feature Feedback, Questions, & Ideas] [answered]
#

**Comment by queenofcorgis** [comment, marked as answer]
Hi @gcs1813, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Comment by queenofcorgis** [comment]
Please note, contacting any member of GitHub Staff via unsolicited mentions or pings is strongly discouraged and may be considered a violation of our prohibition against harassment. Please review our Code of Conduct for more information.

Let's work together to keep GitHub Community a place where people feel safe to participate by being respectful of them and their time.
